src/test/java/org/elasticsearch/transport/ContextAndHeaderTransportTests.java

```diff
@@ -122,7 +122,7 @@ public void testThatTermsLookupGetRequestContainsContextAndHeaders() throws Exce
                 .setSource(jsonBuilder().startObject().field("username", "foo").endObject()).get();
         transportClient().admin().indices().prepareRefresh(queryIndex, lookupIndex).get();
 
-        TermsLookupQueryBuilder termsLookupFilterBuilder = QueryBuilders.termsLookupQuery("username").lookupIndex(lookupIndex).lookupType("type").lookupId("1").lookupPath("followers");
+        TermsQueryBuilder termsLookupFilterBuilder = QueryBuilders.termsLookupQuery("username", lookupIndex, "type", "1", "followers");
```

Comment:
I probably went to far with refactoring here by changing the termsLookupQuery() method signature. Will revert that again.